packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec

```diff
@@ -45,8 +44,13 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %defattr(-,root,root)
 %doc dashboards/README.md
 %license LICENSE
-%attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
-%attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0755,root,root) %dir %{dashboards_dir}/sleha
+%attr(0755,root,root) %config %{dashboards_dir}/sleha/*
+%attr(0755,root,root) %config %{provisioning_dir}/provider-sleha.yaml
```

Comment:
I don't think we need `755` permissions for these files, `644` is probably more appropriate.